### PR TITLE
update OWNERS list

### PR DESCRIPTION
**Pull request by joelddiaz**

### CODE REVIEW

**Review by 2uasimojo — COMMENTED** [review]
sure :)

**Review by akhil-rane — COMMENTED** [review]
/lgtm

**Comment by joelddiaz** [comment]
this is just updating our team membership to reflect current realities.
/approve
/label docs-approved
/label qe-approved
/label px-approved

File: OWNERS [hunk 1]
```diff
@@ -1,9 +1,8 @@
 approvers:
 - abutcher
-- csrwng
-- dgoodwin
+- akhil-rane
+- 2uasimojo
 - joelddiaz
-- twiest
-- staebler
```
Comment (staebler) [inline OWNERS:7]: +1

**Pull request merged**